content/docs/setup/kubernetes/additional-setup/mesh-expansion/index.md

```diff
@@ -288,12 +293,18 @@ which creates a Kubernetes `selector-less` service.
     <body>
     <h2>Directory listing for /</h2>
     <hr>
-    <ul>                                                                
+    <ul>
     <li><a href=".bashrc">.bashrc</a>
     <li><a href=".ssh/">.ssh/</a>
     </body>
     {{< /text >}}
 
+**Congratulations!**
+
+You successfully configured a service running in a pod within the mesh to send
+traffic to an external service running on a VM outside of the mesh and
+tested that the configuration worked.
+
 **Congratulations!**
```

Comment:
Thanks for accepting my suggestion but please remove lines 309-311.

Reply:
Done.